aws-enigma-2/main.tf

```diff
@@ -0,0 +1,205 @@
+EOF
+}
+
+module "aws_az_1b" {
+  source                    = "../modules/aws_workers_az"
+  az_group                  = "1b"
+  bastion_security_group_id = "${data.terraform_remote_state.vpc.bastion_security_group_1b_id}"
+  env                       = "${var.env}"
+  index                     = "${var.index}"
+  vpc_id                    = "${data.terraform_remote_state.vpc.vpc_id}"
+}
+
+module "aws_az_1b2" {
+  source                    = "../modules/aws_workers_az"
+  az_group                  = "1b2"
+  bastion_security_group_id = "${data.terraform_remote_state.vpc.bastion_security_group_1b_id}"
+  env                       = "${var.env}"
+  index                     = "${var.index}"
+  vpc_id                    = "${data.terraform_remote_state.vpc.vpc_id}"
+}
+
+module "aws_az_1e" {
+  source                    = "../modules/aws_workers_az"
+  az_group                  = "1e"
+  bastion_security_group_id = "${data.terraform_remote_state.vpc.bastion_security_group_1e_id}"
+  env                       = "${var.env}"
+  index                     = "${var.index}"
+  vpc_id                    = "${data.terraform_remote_state.vpc.vpc_id}"
+}
+
+module "aws_az_1e2" {
+  source                    = "../modules/aws_workers_az"
+  az_group                  = "1e2"
+  bastion_security_group_id = "${data.terraform_remote_state.vpc.bastion_security_group_1e_id}"
+  env                       = "${var.env}"
+  index                     = "${var.index}"
+  vpc_id                    = "${data.terraform_remote_state.vpc.vpc_id}"
+}
+
+module "aws_asg_org" {
+  source                     = "../modules/aws_asg"
+  cyclist_auth_token         = "${random_id.cyclist_token_org.hex}"
+  cyclist_version            = "v0.5.0"
+  docker_storage_dm_basesize = "19G"
+  env                        = "${var.env}"
+  env_short                  = "${var.env}"
+  github_users               = "${var.github_users}"
+  heroku_org                 = "${var.aws_heroku_org}"
+  index                      = "${var.index}"
+  registry_hostname          = "${data.terraform_remote_state.vpc.registry_hostname}"
+
+  security_groups = [
+    "${module.aws_az_1b.workers_org_security_group_id}",
+    "${module.aws_az_1b2.workers_org_security_group_id}",
+    "${module.aws_az_1e.workers_org_security_group_id}",
+    "${module.aws_az_1e2.workers_org_security_group_id}",
+  ]
+
+  site                                   = "org"
+  syslog_address                         = "${var.syslog_address_org}"
+  worker_ami                             = "${data.aws_ami.tfw.id}"
+  worker_asg_max_size                    = 20
+  worker_asg_min_size                    = 1
+  worker_asg_namespace                   = "Travis/org"
+  worker_asg_scale_in_threshold          = 80
+  worker_asg_scale_in_evaluation_periods = 3
+  worker_asg_scale_in_period             = 300
+  worker_asg_scale_out_threshold         = 60
+  worker_asg_scale_out_qty               = 2
```

Comment:
Yep, the more I look at this, I'm thinking this should be a fixed size ASG neutral or non-existent scaling policies.